packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -336,6 +336,14 @@ def test_metrics_containers_nan(dcos_api_session):
         assert metric_value == '', 'unexpected metric value: {}'.format(metric_value)
 
 
+@retrying.retry(wait_fixed=(2 * 1000), stop_max_delay=(150 * 1000))
+def test_zookeeper_metrics_collected(dcos_api_session):
+    for master in dcos_api_session.masters:
+        response = dcos_api_session.get('/metrics', scheme='http', host=master, port=61091)
+        assert response.status_code == 200, 'Status code: {}'.format(response.status_code)
```

Comment:
❓ I think `pytest`'s assertion will be clear enough without the explanation

Reply:
It probably is, I just copied it from the other tests in the same file. Removing that.